.travis.yml

```diff
@@ -12,7 +9,7 @@ install:
 before_script:
   - npm install -g angular-cli codecov
 script:
-  - npm build
+  - npm run build
```

Comment:
On the old build we decided to do both

npm run build --aot --prod
and 
npm run build

Basic there was errors with a prod build vs dev build (tree shaking and such). Might be a good idea to let that run here as well?

Reply:
Testing both in a row like before though, would give you:
```
npm build  =>          // sohoxi.js = cat
npm build --prod =>    // sohoxi.js = dog
npm lint              // linting dog
npm test              // testing dog
```
...so you'd never actually test cat. If anything we'd have to do:
```
script:
  npm build
  npm run lint
  npm test
  npm build -aot --prod
  npm run lint
  npm test
```

Its up to you?

Reply:
ℹ️ After a slack chat, @tmcconechy and I decided we should test the prod build to avoid any sort of tree-shaking oddities like the team saw before 👍